### PR TITLE
Add capability to list more than one method to be verbose

### PR DESCRIPTION
**Pull request by migueldeicaza**
document some internal APIs, update man page with more explanations.

Feature request: https://github.com/mono/mono/issues/6399

### CODE REVIEW

**Comment by akoeplinger** [comment]
@monojenkins build

File: man/mono.1 [hunk 1]
```diff
@@ -65,6 +65,45 @@ and
 See the description for 
 .B MONO_IOMAP
 in the environment variables section for more details.
+.SH METHOD DESCRIPTIONS
+A number of diagnostic command line options take as argument a method
+description.   A method description is a textual representation that
```
Comment (jaykrell) [inline man/mono.1]: Modern form is one space after period, but I admit I've never written a man page, perhaps they have different conventions, or there was a desire to stretch toward equal length lines?

File: man/mono.1 [hunk 2]
```diff
@@ -65,6 +65,45 @@ and
...
+Generic values should use '<', one or more type names, separated by
+both a comma and a space and '>'.
+.PP
+By-reference arguments should include a "&" after the typename.
+.PP
+Examples:
+.nf
+*:ctor(int)		// All constructors that take an int as an argument
+*:Main			// Methods named Main in any class
+*:Main(string[])	// Methods named Main that take a string array in any class
```
Comment (jaykrell) [inline man/mono.1]: Add a generic example?

Comment (jaykrell) [reply man/mono.1]: eh, also no pointer example or class::method, the text is very clear as to the form, so examples are just being really exceedingly nice, plenty good enough.

**Comment by luhenry** [comment]
@monojenkins build Linux AArch64 Interpreter

**Pull request merged**